cob_description/urdf/drive_wheel/drive_wheel.urdf.xacro

```diff
@@ -42,22 +55,33 @@
 
     <!-- extensions -->
     <xacro:cob_wheel_gazebo parent="${parent}" suffix="${suffix}" />
-    <xacro:cob_wheel_transmission parent="${parent}" suffix="${suffix}" reflect="${reflect}" />
+    <xacro:unless value="${passive}">
+      <xacro:cob_wheel_transmission parent="${parent}" suffix="${suffix}" reflect="${reflect}" />
```

Comment:
no transmission for passive wheel = no controller for passive wheel (also, no joint_states!)

Reply:
no joint_states is OK for passive wheels as we won't have that on real HW too.